src/WebApp.Web/Base/ExceptionHandlingMiddleware.cs

```diff
@@ -24,13 +24,15 @@ private async Task HandleCustomValidationException(HttpContext httpContext, Cust
     {
         var response = new
         {
+            type = "https://tools.ietf.org/html/rfc9110#section-15.5.1",
             title = "One or more validation errors occurred.",
-            status = StatusCodes.Status422UnprocessableEntity,
-            errors = exception.ValidationMessages
+            status = StatusCodes.Status400BadRequest,
+            errors = exception.ValidationMessages,
+            traceId = httpContext.TraceIdentifier,
         };
         
         httpContext.Response.ContentType = "application/json";
-        httpContext.Response.StatusCode = StatusCodes.Status422UnprocessableEntity;
+        httpContext.Response.StatusCode = StatusCodes.Status400BadRequest;
```

Comment:
let's keep 422 to differentiate DataAnnotation-based (schema) validation and FluentValidation-based (logic) validation.